cs/quickstart/getting-started.texy

```diff
@@ -17,7 +17,7 @@ Nette Framework můžeme stáhnout manuálně, ale doporučená cesta je začít
 S Composerem si můžeme velmi jednoduše stáhnout a nainstalovat základní skeleton aplikace, známý jako Web Project, který již obsahuje Nette Framework. Abychom mohli založit projekt, najdeme si kořenový adresář webového serveru (např. `/var/www` nebo `C:\InetPub`) a pomocí příkazové řádky a spustíme následující příkaz.
 
 ```shell
-composer create-project nette/web-project nette-blog
+composer create-project nette/sandbox nette-blog
```

Comment:
Hele to snad nemyslíš vážně… Na to je navázané úplně všechno ostatní, včetně hotového příkladu na GitHubu.

Reply:
ok, vrátím to